_reports/ksdme.md

```diff
@@ -0,0 +1,403 @@
+---
+categories:
+  - GSoC
+  - 2018
+  - StatusReport
+date: 2018-08-12
+student: Kilari Teja
+organisation: coala
+organisation_link: https://coala.io
+report_year: 2018
+project: coala language server
+project_link: https://summerofcode.withgoogle.com/projects/#4971369184362496
+tarball: https://github.com/ksdme/GSoCReport/raw/master/ksdme_coala.tar.gz
+sha256sum: 71985d45fbf7648029ac14541b7012d5ff51bcc0e9b2ebf7032a3f69f3ce5b1a
+mentors: >
+  [Ce Gao](https://github.com/gaocegege) &
+  [Nicholas Gates](https://github.com/gatesn) &
+  [Nalin Bhardwaj](https://github.com/nalinbhardwaj)
+phase:
+  - Bonding: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/47
+  - Phase 1: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/48
+  - Phase 2: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/49
+  - Phase 3: https://gitlab.com/coala/GSoC/GSoC-2018/milestones/50
+bio: >
+  I am Kilari Teja, a final year student of Computer Science Engineering at
+  Guru Gobind Singh Indraprastha University, located in Delhi, India. I have
+  been a GSoC'18 student for [coala](http://coala.io) working on bringing
+  coala to code editors via language server protocol. My projects makes it
+  possible for linting and fixing code to be performed right from the code
+  editor using coala.
+social:
+  - GitHub:
+      - username: ksdme
+      - link: https://github.com/ksdme
+  - GitLab:
+      - username: ksdme
+      - link: https://gitlab.com/ksdme
+  - Gitter:
+      - username: ksdme
+      - link: https://gitter.im/ksdme
+  - OpenHub:
+      - username: ksdme
+      - link: https://www.openhub.net/accounts/ksdme
+email: ksdme9@gmail.com
+blog: http://medium.com/ksdme-coala-and-gsoc
+activity:
+
+  - 1:
+      - repo: projects
+      - link: https://github.com/coala/projects/commit/10d706aa14325147caacd053e20c4fb77af56376
+      - details: >
+         [Bonding Period Work] coala_language_server.md: Update project details
+
+  - 2:
+      - repo: cEPs
+      - link: https://github.com/coala/cEPs/pull/171
+      - details: >
+         cEP-0028: Optimizing coala for language server
+
+  - 3:
+      - repo: coala-vs-code
+      - link: https://github.com/coala/coala-vs-code/pull/32/commits/bd12657c67b257ed4b0f498c14cab389f96cc91a
```

Comment:
use https://github.com/coala/coala-vs-code/commit/bd12657c67b257ed4b0f498c14cab389f96cc91a to be consistent with the others.